### ISSUE TITLE
Create of VLAN using the API and /available-vlans/ fails when role or tenant is present

### ISSUE THREAD

**Issue by einaram** [netbox-community/netbox#13440]
### NetBox version

v3.6-beta1

### Python version

3.10

### Steps to Reproduce

1. Create a tenant and note the ID
2. Create a VLAN group and note the ID
3. Run a POST to `api/ipam/vlan-groups/<VLAN-group-id>/available-vlans/ like:
```

curl -X 'POST' \
  'https://demo.netbox.dev/api/ipam/vlan-groups/2/available-vlans/' \
  -d '{
  "name": "a_VLAN_name",
  "tenant": 2
}'
```

### Expected Behavior

A new VLAN should be created

### Observed Behavior

Bad request is returned with an error message that indicates that the deserialization used the tenant object instead of the ID:

```{
  "tenant": [
    "Related objects must be referenced by numeric ID or by dictionary of attributes. Received an unrecognized value: Strickland Propane"
  ]
}
```

**Comment by einaram** [comment]
I have tested possible solution by not deserializing `requested_vlans` and instead adding `vid` to the raw data. This, and tests modified to test this behavoir is available here:
https://github.com/netbox-community/netbox/compare/develop...einaram:netbox:13440-fix-available-vlan-post

**Comment by jeremystretch** [comment]
@arthanson this may be related to #13746

**Comment by arthanson** [comment]
This is fixed by #13746

**Comment by jeremystretch** [comment]
Confirmed that this has been resolved by PR #13889. Thanks @arthanson!
